docs/toc.yml

```
@@ -402,6 +402,9 @@ items:
       displayName: azure container apps,aca,azd
     - name: Deploy using azd and CI/CD
       href: deployment/azure/aca-deployment-github-actions.md
+    - name: Customize deployments with infrastructure synthesis
+      href: deployment/azure/customize-deployments.md
+      displayName: azd infra synth,bicep,infrastructure,production
```

I'm adding `azd infra gen` but intentionally leaving `azd infra synth` for discoverability.

```suggestion
      displayName: azd infra synth,azd infra gen,bicep,infrastructure,production
```